angular/projects/public-nrpti/src/app/records/records-list/records-list.component.ts

```diff
@@ -215,6 +253,80 @@ export class RecordsListComponent implements OnInit, OnDestroy {
     this.submit();
   }
 
+  keywordSearch() {
+    console.log('&&&', this.keywordSearchWords);
```

Comment:
Remove console logs here and everywhere else.

Reply:
removed